specs/OAuth/OAuth2Manager.md

```diff
@@ -572,17 +483,8 @@ namespace Microsoft.Security.Authentication.OAuth
         // parameters as well as a redirect URI, which is frequently specified.
         static AuthRequestParams CreateForAuthorizationCodeRequest(String clientId, Windows.Foundation.Uri redirectUri);
 
-        // Helper method to create for an implicit grant request ("token" response type) with required parameters, per
-        // RFC 6749 section 4.2.1.
-        static AuthRequestParams CreateForImplicitRequest(String clientId);
-
-        // Helper method to create for an implicit grant request ("token" response type) with required parameters as
-        // well as a redirect URI, which is frequently specified.
-        static AuthRequestParams CreateForImplicitRequest(String clientId, Windows.Foundation.Uri redirectUri);
-
         // Specifies the required "response_type" parameter of the authorization request. This property is initialized
-        // by the creation function used ("code" for 'CreateForAuthorizationCodeRequest' and "token" for
-        // 'CreateForImplicitRequest').
+        // by the creation function used ("code" for 'CreateForAuthorizationCodeRequest').
         //
         // Defined by RFC 6749: The OAuth 2.0 Authorization Framework, sections 4.1.1 and 4.2.1
         //      https://www.rfc-editor.org/rfc/rfc6749#section-4.1.1
```

Comment:
Seems there are some references of `implicit request` in `AuthResponse` class, Shouldn't we remove them?

Reply:
Yes, done. 
Thanks!